streaming/README.md

```diff
@@ -511,4 +511,100 @@ metrics, following the same pattern of the receiving side.
 
 For a practical example see [Monitoring ephemeral nodes](#monitoring-ephemeral-nodes).
 
+## Troubleshooting streaming connections
+
+Sometimes when you are configuring master and slave nodes you can have some errors in your log files, in this section it
+is described the most common errors.
+
+### Connection between slave and master is slow
+
+When you have a slow connection between master and slave, Netdata can raise different errors related to this problem.
+The slave Netdata will log a majority of error messages.
+
+```
+netdata ERROR : STREAM_SENDER[SLAVE HOSTNAME] : STREAM SLAVE HOSTNAME [send to MASTER IP:MASTER PORT]: too many data pending - buffer is X bytes long,
+Y unsent - we have sent Z bytes in total, W on this connection. Closing connection to flush the data.
+```
+
+The the value X, Y, Z and W are integer numbers.
+
+On the other hand, in the master side you can have different reports where the most common message that you can see is
+
+```
+netdata ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : read failed: end of file
+```
+
+Another common problem is the slave sends only part of the message for the master. In this case, the master
```

Comment:
Another common problem in slow connections is the slave sending a partial the message to the master. In this case, the master will write the following inside `error.log`: